src/cmdlib.sh

```diff
@@ -276,6 +276,16 @@ baseurl=file://${workdir}/overrides/rpm
 gpgcheck=0
 EOF
     fi
+    rootfs_overlay="${overridesdir}/rootfs"
+    if [ -d "${rootfs_overlay}" ]; then
+        echo "Committing ${rootfs_overlay}"
+        sudo ostree commit --repo="${workdir}/tmp/repo" --tree=dir="${rootfs_overlay}" -b cosa-bin-overlay \
+          --owner-uid 0 --owner-gid 0 --no-xattrs --no-bindings --parent=none
```

Comment:
Nice, simple enough! And... ahh OK right, rpm-ostree uses the content checksum so the hash changing each time isn't an issue.

Reply:
Yep, I thought about that :smile: